lib/ui/pages/home_page/home_page.dart

```diff
@@ -12,23 +15,41 @@ class HomePage extends StatefulWidget {
 
 class _HomePageState extends State<HomePage> {
   @override
+  int _selectedIndex = 0;
+  final List<Widget> _pages = [
+    const SessionPage(),
+    const SettingsPage(),
+  ];
   void initState() {
     super.initState();
 
     final prefs = GetIt.instance.get<Prefs>();
     prefs.buckets = 1;
   }
 
+  void _onDestinationSelected(int index) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
-        title: const Text('Flow Fusion'),
+        body: Row(children: [
+      SidebarWidget(
+        menuIcons: const [Icons.schedule, Icons.settings],
+        menuLabels: const ['Session', 'Settings'],
+        selectedIndex: _selectedIndex,
+        onDestinationSelected: _onDestinationSelected,
       ),
-      body: Center(
-        child: Assets.images.bucket.image(),
+      const VerticalDivider(thickness: 1, width: 1),
+      // Main content area
+      Expanded(
+        child: Center(
+          child: _pages.elementAt(_selectedIndex),

```

Comment:
https://dart.dev/language/branches#switch-expressions

Reply:
For what we need switch if we have index for rout